### ISSUE TITLE
Erroneous semicolon appearing in combined render

### ISSUE THREAD

**Issue by novalex** [jfcherng/php-diff#30]
When using the combined rendering type, found a strange bug which causes an extra semicolon to appear in the diff even though it's not there in the input strings.

Narrowed the before/after strings that cause this bug to appear to the following:
Before:
```html
<span><a href="#">Something</a>
</span>

<em><span>Something</span></em>
```
After:
```html
<a href="#">Something</a>


<em>Something</em>
```
The options:
```php
$diffOptions   = array(
	'context'          => 5,
	'ignoreWhitespace' => false,
);
$renderOptions = array(
	'detailLevel'   => 'word',
	'separateBlock' => true,
	'language'      => 'eng',
);
```
The rendered diff output:
<img width="1041" alt="Screenshot 2020-04-24 at 15 55 24" src="https://user-images.githubusercontent.com/10091929/80226874-d5239b00-8644-11ea-8f2e-a3d96114b29e.png">

Removing the newline between the tags will cause the semicolon position to shift:
<img width="1042" alt="Screenshot 2020-04-24 at 16 03 49" src="https://user-images.githubusercontent.com/10091929/80227176-35b2d800-8645-11ea-9b92-18ce56a88d5d.png">

**Comment by jfcherng** [comment]
Thanks for the reproducer. I will look into it when I am available.

Temporarily, I further simplify it with the following which should be more feasible to debug it.


**old**
```
<
```

***new**
```


A<B
```

**options**
```php
$renderOptions = [
    'detailLevel'   => 'word',
];
```


![image](https://user-images.githubusercontent.com/6594915/80229920-bbee0f00-8683-11ea-92e8-dd01cb28b5bb.png)

**Comment by jfcherng** [comment]
This should've been fixed in `6.7.2` :smiley:
